core/src/global.rs

```diff
@@ -187,8 +187,11 @@ where
 {
 	let chain_type = get_chain_type();
 	match chain_type {
-		// Mainnet has Cuckaroo(d)29 for AR and Cuckatoo31+ for AF
+		// Mainnet has Cuckaroo{,d,m,z}29 for AR and Cuckatoo31+ for AF
```

Comment:
`{,`

Reply:
The original AR PoW was simply called Cuckaroo, so the first of the 4 options has an empty string suffix, hence the {,